web/app/configs/lang.json

```diff
@@ -535,8 +535,8 @@
             "EH": "Sahara Occidentale",
             "ER": "Eritrea",
             "ES": "Spagna",
-            "ES-CN": "Spagna (Isole Canarie)",
```

Comment:
To be consistent I think it is better to prefix with the country (as we do in Canada for example)